build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1301,9 +1308,13 @@ message GetCapabilitiesRequest {
 // [Capabilities.GetCapabilities][build.bazel.remote.execution.v2.Capabilities.GetCapabilities].
 message ServerCapabilities {
   // Capabilities of the remote cache system.
+  // This field will be returned by endpoints that support CAS and Action
+  // Cache services.
```

Comment:
Does this comment add additional value? If not, could we remove it? Same below.

Reply:
Sure.